Content.Server/Store/Systems/StoreSystem.cs

```diff
@@ -69,10 +70,13 @@ private void OnStoreOpenAttempt(EntityUid uid, StoreComponent component, Activat
         if (!component.OwnerOnly)
             return;
 
-        component.AccountOwner ??= args.User;
+        if (!_mind.TryGetMind(args.User, out var mind, out var mindComp))
+            return;
+
+        component.AccountOwner ??= new Entity<MindComponent?>(mind, mindComp);
```

Comment:
```suggestion
        component.AccountOwner ??= (mind, mindComp);
```

You dont need to make a new Entity, you can just make a tuple.
Might need another maint to sanity check me on this.

Reply:
The tuple gets cast to entity anyway so entity is probably better.